cypress/e2e/title-level-request/user-can-see-move-request-button-in-actions-for-request-details.cy.js

```diff
@@ -0,0 +1,88 @@
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import Requests from '../../support/fragments/requests/requests';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import TitleLevelRequests from '../../support/fragments/settings/circulation/titleLevelRequests';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import UserEdit from '../../support/fragments/users/userEdit';
+import { FULFILMENT_PREFERENCES, REQUEST_LEVELS, REQUEST_TYPES } from '../../support/constants';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import RequestDetail from '../../support/fragments/requests/requestDetail';
+
+describe('Title Level Request', () => {
+  const testData = {
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    folioInstances: InventoryInstances.generateFolioInstances(),
+  };
+  let requestId;
+
+  before('Create test data', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+    Location.createViaApi(testData.defaultLocation).then((location) => {
+      InventoryInstances.createFolioInstancesViaApi({
```

Comment:
Use createFolioInstanceViaApi. You need only one

Reply:
Done